modules/siddhi-query-api/src/main/java/org/wso2/siddhi/query/api/definition/TableDefinition.java

```diff
@@ -44,6 +44,17 @@ public TableDefinition annotation(Annotation annotation) {
         return this;
     }
 
+    public void replaceAnnotation(Annotation annotation) {
```

Comment:
Shall we remove this method and add another annotation instead of replacing the one there.